vault/gcp.go

```diff
@@ -0,0 +1,96 @@
+package vault
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/hashcode"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type GCPBinding struct {
```

Comment:
I think it would be good to add some documentation here.

Reply:
Added something simple.